components/Comment/lib/types.ts

```diff
@@ -0,0 +1,53 @@
+import { parseCreatedBy } from "~/config/types/contribution";
+import { CreatedBy, ID } from "~/config/types/root_types";
+import { formatDateStandard, timeSince } from "~/config/utils/dates";
+
+export enum POST_TYPES {
```

Comment:
Seems like an opportunity to rely on a common definition with https://github.com/ResearchHub/researchhub-web/blob/35cdceb27280516a454ccd8736b5fb52a38cf4f9/components/TextEditor/config/postTypes.ts#L15.

Reply:
I appreciate the thoroughness in the PR and noticing there are two similar type defs. You are absolutely right!

What I have been doing here is copying all the good stuff from the "old comment" system into then new `Comment/*` comment system because the intention is to delete the old one once the new one is ready. This means that we will temporarily have a few duplicate things here and there